components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsMessageTable/src/EIAnalyticsMessageTable.jsx

```diff
@@ -121,8 +121,15 @@ class EIAnalyticsMessageTable extends Widget {
         super.subscribe(this.handlePublisherParameters);
     }
 
-    handlePublisherParameters(message) {
-        if ('granularity' in message) {
+    handlePublisherParameters(recievedMessage) {
+        let message;
```

Comment:
Lets reduce the following block into a one line
`const message = (typeof recievedMessage === 'string') ? JSON.parse( ) : recievedMessage;`

Reply:
Fixed at 232cc7dff2df7a50828020f9fca1dac61295ef44